Sources/Rendering/Misc/SynchronizableRenderWindow/vtkObjectManager.js

```diff
@@ -290,8 +289,8 @@ function polydataUpdater(instance, state, context) {
       .then(
         (values) => {
           const array = vtkDataArray.newInstance(Object.assign({ values }, arrayMetadata));
-          // console.log('array', array.getName());
-          arraysToBind.push([instance.get(arrayMetadata.location)[arrayMetadata.location].addArray, [array]]);
+          const regMethod = arrayMetadata.registrationMethod ? arrayMetadata.registrationMethod : 'addArray';
```

Comment:
I'd almost rather change that dataset reader to say `registrationMethod`, but up to you...